apps/system/js/bluetooth_transfer.js

```diff
@@ -62,6 +62,7 @@ var BluetoothTransfer = {
       this.sendFileViaHandover.bind(this));
 
     Service.registerState('isSendFileQueueEmpty', this);
+    Service.registerState('isFileTransferInProgress', this);
```

Comment:
maybe adding a unittest case with Service.registerState.calledWith('isFileTransferInProgress') helps followers not breaking it?